ci/start_init_test.sh

```diff
@@ -74,16 +80,81 @@ if [ "$x"!="" ]; then
         fi
 
         # TEST#3: Run the libiscsi compliance suite on Jiva Vol
-        sudo mkdir /mnt/logs 
+        sudo mkdir /mnt/logs
         sudo docker run -v /mnt/logs:/mnt/logs --net host openebs/tests-libiscsi /bin/bash -c "./testiscsi.sh --ctrl-svc-ip 172.18.0.2"
         tp=$(grep "PASSED" $(find /mnt/logs -name SUMMARY.log) | wc -l)
         tf=$(grep "FAILED" $(find /mnt/logs -name SUMMARY.log) | wc -l)
         if [ $tp -ge 146 ] && [ $tf -le 29 ]; then
-            echo "iSCSI Compliance test: PASSED"
-        else 
+           echo "iSCSI Compliance test: PASSED"
+        else
             echo "iSCSI Compliance test: FAILED"; exit 1
         fi
-else 
+else
         echo "Unable to detect iSCSI device, login failed"; exit 1
-fi 
+fi
+
+sudo umount /mnt/store
+sudo iscsiadm -m node -u
+sudo iscsiadm -m node -o delete
+
+# Test clone feature
+sudo docker run -d --net stg-net --ip 172.18.0.5 -P --expose 3260 --expose 9501 --expose 9502-9504 $JI launch controller --frontend gotgt --frontendIP 172.18.0.5 store1
+sudo docker run -d -it --net stg-net --ip 172.18.0.6 -P --expose 9502-9504 -v /tmp/vol3:/vol3 $JI launch replica --type clone --snapName snap1 --cloneIP 172.18.0.2 --frontendIP 172.18.0.5 --listen 172.18.0.6:9502 --size 2g /vol3
+
+clonestatus=`curl http://172.18.0.6:9502/v1/replicas/1 | jq '.clonestatus' | tr -d '"'`
+
+i=0
+while [ -z $clonestatus ]; do
+        sleep 3
+	clonestatus=`curl http://172.18.0.6:9502/v1/replicas/1 | jq '.clonestatus' | tr -d '"'`
+        i=`expr $i + 1`
+        if [ $i -eq 20 ]; then
```

Comment:
Why not run until we get an error, after all, we added error cases yes?

Reply:
We are exiting the container if the replica clone fails. Therefore we will and be able to get the status of the replica and this will be stuck in infinite loop. Or we could also check for the http error.